scripts/create_and_push_tag_from_package_json_version.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+
+create_and_push_tag_from_package_json_version(){
+  VERSION=`yarn --silent json -f package.json version`
+  git tag -a "$1${VERSION}" -m "v${VERSION}"
+  git push origin "$1${VERSION}"
```

Comment:
lorsqu'un script ou une fonction prends des arguments
je recommande de faire des variables d'alias
pour une mieux révéler l'intention de la valeur des arguments attendus


```suggestion
  TAG_PREFIX="$1"
  git tag -a "${TAG_PREFIX}${VERSION}" -m "v${VERSION}"
  git push origin "${TAG_PREFIX}${VERSION}"
```